extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardResourceStatusPage.ts

```diff
@@ -113,6 +140,12 @@ export class BdcDashboardResourceStatusPage {
 			return;
 		}
 
+		this.lastUpdatedLabel.updateProperty('value',
```

Comment:
do you have to call the method instead of this.lastUpdatedLabel.value = ?

Reply:
Oh hey that's much easier!